crates/mirror-mirror/src/lib.rs

```diff
@@ -196,7 +195,6 @@
 //!
 //! Name | Description | Default?
 //! ---|---|---
-//! `std` | Enables using the standard library (`core` and `alloc` are always required) | Yes
```

Comment:
Wanna mention the new `simple_type_name` feature here?

Reply:
Ah, yeah